### PR TITLE
FOIA-1189304: Security Update hotfix.

### PR DESCRIPTION
**Pull request by jhuang0427**

### CODE REVIEW

**Comment by brockfanning** [comment]
@jhuang0427 It looks like there is an updated patch: https://www.drupal.org/files/issues/2021-04-06/corrupted_data_paragraphs-3060223-46.patch

I found that on the issue page: https://www.drupal.org/node/3060223

Did you try using this one?

**Comment by jhuang0427** [comment]
I have committed to PR with alternative on drupal/entity_clone

File: composer.json [hunk 1]
```diff
@@ -125,6 +125,9 @@
             },
             "drupal/entity_clone": {
```
Comment (brockfanning) [inline composer.json:126]: @jhuang0427 We only need one "drupal/entity_clone" entry - you can remove the old one and just have the new one.

**Comment by jhuang0427** [comment]
have removed older one.

**Pull request merged**